semanticdb-javac/src/main/java/com/sourcegraph/semanticdb_javac/GlobalSymbolsCache.java

```diff
@@ -32,12 +35,16 @@ public String semanticdbSymbol(Symbol sym, LocalSymbolsCache locals) {
     return result;
   }
 
+  public boolean isNone(TypeMirror mirror) {
+    return mirror == null || mirror.getKind() == TypeKind.NONE;
+  }
+
   public boolean isNone(Symbol sym) {
-    return sym == null || sym.kind == Kinds.NIL || (sym.kind & Kinds.ERRONEOUS) != 0;
+    return sym == null; // || sym.kind == Kinds.Kind.NIL || sym.kind == Kinds.Kind.ERR;
```

Comment:
This seems to be working ok. No? I'm OK with removing the comment.

Reply:
The commented part is not backwards compatible. JDK 8 it was `Kinds.NIL` etc, JDK 9+ the enum was moved to be a nested enum. The behaviour appears to be identical at best, but I'm unsure if we're testing enough edge cases to surface where this change may not be sufficient